tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920290.yaml

```diff
@@ -21,31 +21,3 @@ tests:
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
           output:
             status: [403, 400]
-            #log_contains: "id \"920290\""
-
-#-
-#test_title: 920290-2
-#stages:
-#  -
-#    stage:
-#      input:
-#        dest_addr: "127.0.0.1"
-#        port: 80
-#        headers:
-#            User-Agent: "ModSecurity CRS 3 Tests"
-#            Host: "%00"
-#      output:
-#        no_log_contains: "id \"920290\""
-#    -
-#  test_title: 920290-3
-#  stages:
-#    -
-#      stage:
-#        input:
-#          dest_addr: "127.0.0.1"
-#          port: 80
-#          headers:
-#              User-Agent: "ModSecurity CRS 3 Tests"
-#              Host: "localhost"
-#        output:
-#          no_log_contains: "id \"920290\""
```

Comment:
These have been disabled since https://github.com/coreruleset/coreruleset/commit/3097bd1f61ef55bd36a0883d24d47505429f078e#diff-394fb9bb413bdd558ba503d654286a6cef9de321478c53c243248a154f3ec710. Do you know why? Maybe we could re-enable them instead of removing them?

Reply:
Went back trying to know why, but couldn't figure it out. Enabling because they look like passing. 🤷